### PR TITLE
Document required Gradle properties

### LINKED ISSUE #4: Document required Gradle properties
I'm starting to use this on a personal project and it took some digging to figure out the names of the properties that Startship expects in gradle.properties. The error message also mentions the option of using `-c` but doesn't say the format of the arguments.

I'd be happy add either or both of these to the readme if you're open to it.

### PR DESCRIPTION
**Pull request by drewhamilton**
Closes #4.

Adds documentation to readme for expected GROUP, POM_ARTIFACT_ID, VERSION_NAME, SONATYPE_NEXUS_USERNAME, and SONATYPE_NEXUS_PASSWORD properties.

Adds "`-c group:artifact:version`" example to readme and to command-line error message. Adds tests related to the updated error message.

### CODE REVIEW

File: src/test/kotlin/nevam/MavenCoordinatesTest.kt [hunk 1]
```diff
@@ -18,4 +21,18 @@ class MavenCoordinatesTest {
...
+
+  @Test fun `read properties from file with all properties succeeds`() {
+    val coordinates = MavenCoordinates.readFrom("src/test/resources/full.gradle.properties")
+    assertThat(coordinates.toString()).isEqualTo("com.squareup.sqldelight:runtime:1.4.0")
+  }
+
+  @Test fun `read properties from file with missing properties suggests -c`() {
+    try {
+      MavenCoordinates.readFrom("src/test/resources/missing.gradle.properties")
+      fail("Expected a CliktError suggesting -c")
```
Comment (saket) [inline src/test/kotlin/nevam/MavenCoordinatesTest.kt]: Can `assertThrows` be used here instead of using an exception as a failsafe?

**Review by saket — CHANGES_REQUESTED** [review]
This looks great. Thanks a bunch!

File: src/test/kotlin/nevam/MavenCoordinatesTest.kt (hunk 1 above)
Comment (drewhamilton) [reply src/test/kotlin/nevam/MavenCoordinatesTest.kt]: I'm not familiar with `assertThrows`. Googling tells me it's in JUnit 5 but this project uses JUnit 4. Is there another library that has it that you want me to use?

Alternatively I could do `@Test(expected = CliktError::class)` but then I can't verify the message.

Happy to do it whatever way you prefer, let me know!

Comment (saket) [reply src/test/kotlin/nevam/MavenCoordinatesTest.kt]: Ah right. I should try out junit 5 for startship but that's outside the scope. What do you think of using `ExpectedException` rule here? If it's too much of an effort then let's go ahead with this for now.

Comment (drewhamilton) [reply src/test/kotlin/nevam/MavenCoordinatesTest.kt]: Done!

**Review by saket — APPROVED** [review]

**Comment by saket** [comment]
🚀

**Comment by saket** [comment]
🚢

**Pull request merged**